metadata/src/main/java/org/apache/kafka/image/writer/ImageWriterOptions.java

```diff
@@ -45,6 +46,7 @@ public Builder setMetadataVersion(MetadataVersion metadataVersion) {
             if (metadataVersion.isLessThan(MetadataVersion.MINIMUM_BOOTSTRAP_VERSION)) {
                 // When writing an image, all versions less than 3.3-IV0 are treated as 3.0-IV1.
                 // This is because those versions don't support FeatureLevelRecord.
+                setOrgMetadataVersion(metadataVersion);
                 setRawMetadataVersion(MetadataVersion.MINIMUM_KRAFT_VERSION);
             } else {
                 setRawMetadataVersion(metadataVersion);
```

Comment:
Let's set both metadata versions here as well so we don't have to deal with a `null` later

Reply:
Sure. I've set both fields in this function call. Note that I've also modified the `ImageDowngradeTest` to use the `setMetadataVersion` instead of `setRawMetadataVersion` (Maybe we should change `setRawMetadataVersion` to private` also ?)